stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -211,114 +304,194 @@ public fun DefaultMessageContainer(
                 .then(clickModifier)
         ) {
 
-            if (!ownsMessage) {
-                MessageAvatar(position, message.user)
-            }
+            leadingContent(messageItem)
```

Comment:
So here, if we don't own the message, we'll show the empty spacer, but on the trailing content, we can show the avatar (if the users want to do that)?

Essentially we have the avatar fro other people's messages, but nothing for ours.